src/goto-instrument/accelerate/trace_automaton.h

```diff
@@ -69,15 +72,16 @@ class automatont
     init_state=that.init_state;
   }
 
+  static const statet no_state=std::numeric_limits<statet>::max();
```

Comment:
Visual Studio is unhappy about this, will need to be moved to the constructor instead.